example/spec/parallel/ingestGranule/IngestGranuleFtpSuccessSpec.js

```diff
@@ -107,15 +107,23 @@ describe('The FTP Ingest Granules workflow', () => {
     });
 
     it('uploaded the granules with correct ContentType', async () => {
-      console.log(`File object on intermittently failing test: ${JSON.stringify(granule.files)}`);
-      console.log(`Granule object on intermittently failing test: ${JSON.stringify(granule)}`);
-      const headObjects = await Promise.all(granule.files.map(async (fileObject) =>
-        ({
-          ...fileObject,
-          ...await headObject(fileObject.bucket, fileObject.key),
-          expectedMime: mime.lookup(fileObject.key) || 'application/octet-stream'
-        })));
-      headObjects.forEach((headObj) => expect(headObj.expectedMime).toEqual(headObj.ContentType));
+      const objectTests = await pMap(
+        granule.files,
+        async ({ bucket, key }) => {
+          const headObjectResponse = await headObject(
+            bucket, key, { retries: 5 }
```

Comment:
`{ retries: 5 }` is the substantive change here. The rest is just refactoring to make it a little easier to understand what is going on here.

Reply:
Is there anything this sindresorhus guy doesn't do?       The logic change is what makes it more readable, although I did have to read docs on pMap.... which I'm going to immediately use elsewhere.   👍

Reply:
If you think that's cool, check out https://github.com/sindresorhus/promise-fun